extensions/multibindings/src/com/google/inject/multibindings/ProvidesIntoOptional.java

```diff
@@ -50,6 +50,9 @@
 @Target(METHOD)
 @Retention(RUNTIME)
 public @interface ProvidesIntoOptional {
+  /**
+   * @since 4.0
```

Comment:
Oh nevermind it's already there -- we can kill this one I think since it's an inner class on a brand new type.